tests/blackbox/stratis_cli_cert.py

```diff
@@ -180,6 +181,23 @@ def test_filesystem_list_empty(self):
         """
         self.unittest_command([STRATIS_CLI, "filesystem", "list"], 0, True, False)
 
+    def test_key_set_unset(self):
+        """
+        Test setting and unsetting a key.
+        """
+        with RandomKeyTmpFile() as fname:
+            self.unittest_command([STRATIS_CLI, "key", "set", "testkey1", "--keyfile-path", fname], 0, True, False)
+            self.unittest_command([STRATIS_CLI, "key", "unset", "testkey1"], 0, True, False)
```

Comment:
This can be pulled out from under `with RandomKeyTmpFile` because once we've added the key, we no longer need it, just the key description.